drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -2647,7 +2647,7 @@ static int rswitch_add_ipv4_dst_route(struct rswitch_ipv4_route *routing_list, s
 
 static void rswitch_fib_event_add(struct rswitch_fib_event_work *fib_work)
 {
-	struct fib_entry_notifier_info fen;
+	struct fib_entry_notifier_info fen = {0};
```

Comment:
Below I can see `	fen = fib_work->fen_info;`

You can move it there if you wish:
`struct fib_entry_notifier_info fen = fib_work->fen_info;`